src/drivers/imu/bmi055/BMI055_accel.cpp

```diff
@@ -214,48 +214,34 @@ BMI055_accel::probe()
 int
 BMI055_accel::accel_set_sample_rate(float frequency)
 {
-	uint8_t setbits = 0;
-	uint8_t clearbits = BMI055_ACCEL_BW_1000;
-
 	if (frequency < (3125 / 100)) {
-		setbits |= BMI055_ACCEL_BW_7_81;
 		_accel_sample_rate = 1563 / 100;
 
 	} else if (frequency < (625 / 10)) {
-		setbits |= BMI055_ACCEL_BW_15_63;
 		_accel_sample_rate = 625 / 10;
 
 	} else if (frequency < (125)) {
-		setbits |= BMI055_ACCEL_BW_31_25;
 		_accel_sample_rate = 625 / 10;
 
 	} else if (frequency < 250) {
-		setbits |= BMI055_ACCEL_BW_62_5;
 		_accel_sample_rate = 125;
 
 	} else if (frequency < 500) {
-		setbits |= BMI055_ACCEL_BW_125;
 		_accel_sample_rate = 250;
 
 	} else if (frequency < 1000) {
-		setbits |= BMI055_ACCEL_BW_250;
 		_accel_sample_rate = 500;
 
 	} else if (frequency < 2000) {
-		setbits |= BMI055_ACCEL_BW_500;
 		_accel_sample_rate = 1000;
 
 	} else if (frequency >= 2000) {
-		setbits |= BMI055_ACCEL_BW_1000;
 		_accel_sample_rate = 2000;
 
 	} else {
 		return -EINVAL;
 	}
 
-	/* Write accel ODR */
-	modify_reg(BMI055_ACC_BW, clearbits, setbits);
-
 	return OK;
```

Comment:
@bkueng - with the removal of the setting of the register does this all become a NOP?

![image](https://user-images.githubusercontent.com/1945821/52590210-9c0f5a00-2df5-11e9-8c4e-94f31f738fa5.png)